diff -aurN PerlPanel-0.8.0.orig/Makefile PerlPanel-0.8.0/Makefile
--- PerlPanel-0.8.0.orig/Makefile	2004-09-27 12:39:19.000000000 +0200
+++ PerlPanel-0.8.0/Makefile	2004-09-29 22:08:25.454719944 +0200
@@ -88,8 +88,8 @@
 	install -m 0755 build/perlpanel-run-dialog.1.gz		$(DESTDIR)/$(MANDIR)/$(MAN_SECTION)/
 	install -m 0755 build/PerlPanel::MenuBase.3.gz		$(DESTDIR)/$(MANDIR)/$(MAN_LIBS_SECTION)/
 	install -m 0755 build/PerlPanel::DesktopEntry.3.gz	$(DESTDIR)/$(MANDIR)/$(MAN_LIBS_SECTION)/
-	install -m 0644 build/locale/en/$(LC_CATEGORY)/perlpanel.mo $(LOCALEDIR)/en/$(LC_CATEGORY)/
-	install -m 0644 build/locale/de/$(LC_CATEGORY)/perlpanel.mo $(LOCALEDIR)/de/$(LC_CATEGORY)/
+	install -m 0644 build/locale/en/$(LC_CATEGORY)/perlpanel.mo $(DESTDIR)/$(LOCALEDIR)/en/$(LC_CATEGORY)/
+	install -m 0644 build/locale/de/$(LC_CATEGORY)/perlpanel.mo $(DESTDIR)/$(LOCALEDIR)/de/$(LC_CATEGORY)/
 
 	@echo Installing default rcfile:
 	@if [ -e $(DESTDIR)/$(CONFDIR)/perlpanelrc ] ; then \
