--- PerlPanel-0.3.0/Makefile.orig	2004-01-17 23:44:33.000000000 +0100
+++ PerlPanel-0.3.0/Makefile	2004-01-17 23:58:56.000000000 +0100
@@ -18,7 +18,7 @@
 
 VERSION=0.3.0
 
-PREFIX=/usr/local
+PREFIX=/usr
 LIBDIR=$(PREFIX)/lib/perlpanel
 BINDIR=$(PREFIX)/bin
 DATADIR=$(PREFIX)/share
@@ -37,14 +37,14 @@
 	pod2man doc/perlpanel-applet-howto.pod > build/perlpanel-applet-howto.1
 
 install:
-	mkdir -p $(LIBDIR) $(BINDIR) $(MANDIR)/$(MAN_SECTION) $(IMGDIR)
-	install -m 0755 build/perlpanel $(BINDIR)/
-	install -m 0755 build/perlpanel-item-edit $(BINDIR)/
-	install -m 0755 src/perlpaneld $(BINDIR)/
-	cp -Rvp lib/* $(LIBDIR)/
-	install -m 0755 build/perlpanel.1 $(MANDIR)/$(MAN_SECTION)/
-	install -m 0755 build/perlpanel-applet-howto.1 $(MANDIR)/$(MAN_SECTION)/
-	cp -Rvp share/pixmaps/* $(IMGDIR)/
+	mkdir -p $(DESTDIR)$(LIBDIR) $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)$(MAN_SECTION) $(DESTDIR)$(IMGDIR)
+	install -m 0755 build/perlpanel $(DESTDIR)$(BINDIR)/
+	install -m 0755 build/perlpanel-item-edit $(DESTDIR)$(BINDIR)/
+	install -m 0755 src/perlpaneld $(DESTDIR)$(BINDIR)/
+	cp -Rvp lib/* $(DESTDIR)$(LIBDIR)/
+	install -m 0755 build/perlpanel.1 $(DESTDIR)$(MANDIR)$(MAN_SECTION)/
+	install -m 0755 build/perlpanel-applet-howto.1 $(DESTDIR)$(MANDIR)$(MAN_SECTION)/
+	cp -Rvp share/pixmaps/* $(DESTDIR)$(IMGDIR)/
 
 clean:
 	rm -rf build
