diff -aurN PerlPanel-0.3.1.orig/Makefile PerlPanel-0.3.1/Makefile
--- PerlPanel-0.3.1.orig/Makefile	2004-02-02 13:54:20.000000000 +0100
+++ PerlPanel-0.3.1/Makefile	2004-02-02 23:46:56.758754592 +0100
@@ -18,7 +18,7 @@
 
 VERSION=0.3.1
 
-PREFIX=/usr/local
+PREFIX=/usr
 LIBDIR=$(PREFIX)/lib/perlpanel
 BINDIR=$(PREFIX)/bin
 DATADIR=$(PREFIX)/share
@@ -41,17 +41,17 @@
 	pod2man lib/PerlPanel/MenuBase.pm > build/PerlPanel::MenuBase.1
 
 install:
-	mkdir -p $(LIBDIR) $(BINDIR) $(MANDIR)/$(MAN_SECTION) $(MANDIR)/$(MAN_LIBS_SECTION) $(IMGDIR)
-	install -m 0755 build/perlpanel $(BINDIR)/
-	install -m 0755 build/perlpanel-item-edit $(BINDIR)/
-	install -m 0755 build/perlpanel-run-dialog $(BINDIR)/
-	install -m 0755 src/perlpaneld $(BINDIR)/
-	cp -Rvp lib/* $(LIBDIR)/
-	install -m 0644 build/PerlPanel.pm $(LIBDIR)/
-	install -m 0755 build/perlpanel.1 $(MANDIR)/$(MAN_SECTION)/
-	install -m 0755 build/perlpanel-applet-howto.1 $(MANDIR)/$(MAN_SECTION)/
-	install -m 0755 build/PerlPanel::MenuBase.1 $(MANDIR)/$(MAN_LIBS_SECTION)/
-	cp -Rvp share/pixmaps/* $(IMGDIR)/
+	mkdir -p $(DESTDIR)$(LIBDIR) $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/$(MAN_SECTION) $(DESTDIR)$(MANDIR)/$(MAN_LIBS_SECTION) $(DESTDIR)$(IMGDIR)
+	install -m 0755 build/perlpanel $(DESTDIR)$(BINDIR)/
+	install -m 0755 build/perlpanel-item-edit $(DESTDIR)$(BINDIR)/
+	install -m 0755 build/perlpanel-run-dialog $(DESTDIR)$(BINDIR)/
+	install -m 0755 src/perlpaneld $(DESTDIR)$(BINDIR)/
+	cp -Rvp lib/* $(DESTDIR)$(LIBDIR)/
+	install -m 0644 build/PerlPanel.pm $(DESTDIR)$(LIBDIR)/
+	install -m 0755 build/perlpanel.1 $(DESTDIR)$(MANDIR)/$(MAN_SECTION)/
+	install -m 0755 build/perlpanel-applet-howto.1 $(DESTDIR)$(MANDIR)/$(MAN_SECTION)/
+	install -m 0755 build/PerlPanel::MenuBase.1 $(DESTDIR)$(MANDIR)/$(MAN_LIBS_SECTION)/
+	cp -Rvp share/pixmaps/* $(DESTDIR)$(IMGDIR)/
 
 clean:
 	rm -rf build
